### PR TITLE
Update composer_storage_trigger.py script to support stable API

### LINKED ISSUE #6631: Composer script to trigger DAG doesn't work for stable API
## In which file did you encounter the issue?

https://github.com/GoogleCloudPlatform/python-docs-samples/blob/9c71acbdf534defb400212ea68cf8359e3a3c4c3/composer/functions/composer_storage_trigger.py

### Did you change the file? If so, how?

Updated code to support stable API as well (alongside with experimental).

## Describe the issue

Starting from Airflow 2 customers can use stable API which is more complete and has authorization mechanism. In the upcoming Composer release (1.17.0-preview.12), this stable API will be accessible for customers and this PR is adjusting trigger DAG script to use it.

### PR DESCRIPTION
**Pull request by kosteev**
## Description

Fixes #6631

## Checklist
- [X ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

Partial run of tests passed.

### CODE REVIEW

**Comment by kosteev** [comment]
This is ready for review and if approved to be merged.
@leahecole Can you, please, take a look.

**Comment by leahecole** [comment]
Hey @kosteev  - the experimental API part works great - I tested it today, but I am having trouble testing it against an env w/ stable API because creating an env with 1.17.0-preview.12 is failing for me. If you have an env I can use to test it on please send me the details and I'll test ASAP!

**Review by leahecole — APPROVED** [review]
I added a comment about using the experimental vs stable API and a link to the docs. 

Otherwise, now that you unblocked me this morning, this looks great! Thanks for the change!

Googlers, see cl/368290505 for a related docs refresh

File: composer/functions/composer_storage_trigger.py [hunk 1]
```diff
@@ -46,16 +49,22 @@ def trigger_dag(data, context=None):
     webserver_id = 'YOUR-TENANT-PROJECT'
     # The name of the DAG you wish to trigger
     dag_name = 'composer_sample_trigger_response_dag'
+
+    if USE_EXPERIMENTAL_API:
+        endpoint = 'api/experimental/dags/{}/dag_runs'.format(dag_name)
```
Comment (dinagraves) [inline composer/functions/composer_storage_trigger.py:54]: Should these use f-strings?

**Review by dinagraves — APPROVED** [review]

File: composer/functions/composer_storage_trigger.py (hunk 1 above)
Comment (leahecole) [reply composer/functions/composer_storage_trigger.py:54]: probably. I'll test them with f strings now just to check

Comment (leahecole) [reply composer/functions/composer_storage_trigger.py:54]:
```suggestion
        endpoint = f'api/experimental/dags/{dag_name}/dag_runs'
```

File: composer/functions/composer_storage_trigger.py [hunk 2]
```diff
@@ -46,16 +49,22 @@ def trigger_dag(data, context=None):
...
+        json_data = {'conf': data, 'replace_microseconds': 'false'}
+    else:
+        endpoint = 'api/v1/dags/{}/dagRuns'.format(dag_name)
```
Comment (leahecole) [inline composer/functions/composer_storage_trigger.py:57]:
```suggestion
        endpoint = f'api/v1/dags/{dag_name}/dagRuns'
```

**Pull request merged**